clouddriver-artifacts/src/test/java/com/netflix/spinnaker/clouddriver/artifacts/ivy/IvyArtifactProviderPropertiesTest.java

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright 2018 Pivotal, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.netflix.spinnaker.clouddriver.artifacts.ivy;
-
-import com.fasterxml.jackson.databind.ObjectMapper;
-import java.io.IOException;
-import org.junit.jupiter.api.Test;
-
-class IvyArtifactProviderPropertiesTest {
-  @Test
-  void deserializeIvySettings() throws IOException {
```

Comment:
The only thing this test was testing was that we could deserialize the account from JSON, but as far as I can tell we never actually deserialize artifact accounts from JSON.  These are classes meant to hold Spring config data and are initialized by Spring; as `IvyArtifactProviderProperties.class` is package-private nobody outside the package should be deserializing it either.  I suspect this was basically added as a placeholder test to have some coverage for the class.

The test breaks because deserializing immutable classes requires a bunch more Jackson annotations, etc...which I could add but given that there seems to be no need to support this I'd rather not include, so I deleted the test.